src/Transition/CreateNewIDFUsingRulesV8_7_0.f90

```diff
@@ -427,7 +425,7 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                    OutArgs(35) = 'MutuallyExclusive'
                  END IF
 
-              CASE('AIRFLOWNETWORK:DISTRIBUTION:COMPONENT:DUCT')
+             CASE('AIRFLOWNETWORK:DISTRIBUTION:COMPONENT:DUCT')
```

Comment:
Is this comment still valid?  I don't see any out of place diffs here in transition.  Maybe they were here before.